### PR TITLE
fix: customer filter default values for customer group and territory

### LINKED ISSUE #18813: customers default filter
Default customers page filter 
All Customer Groups and All Territories
not listing the sub customers groups such as commercial group

### PR DESCRIPTION
**Pull request by 0Pranav**
fixes: https://github.com/frappe/erpnext/issues/18813

### CODE REVIEW

File: erpnext/selling/doctype/customer/customer_list.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
 frappe.listview_settings['Customer'] = {
 	add_fields: ["customer_name", "territory", "customer_group", "customer_type", "image"],
+
+	onload: function(listview) {
+		frappe.route_options = {
+			"customer_group": "",
```
Comment (nabinhait) [inline erpnext/selling/doctype/customer/customer_list.js:6]: From where those default values are coming?

Comment (0Pranav) [reply erpnext/selling/doctype/customer/customer_list.js:6]: it's set in boot.py https://github.com/frappe/erpnext/blob/develop/erpnext/startup/boot.py#L19

Comment (0Pranav) [reply erpnext/selling/doctype/customer/customer_list.js:6]: from here it's set in `frappe.sys_defaults` so it is inserted in filter with global defaults.

Comment (nabinhait) [reply erpnext/selling/doctype/customer/customer_list.js:6]: I have removed default territory from Selling Settings, cleared cache and reloaded the system. Still default territory is set in Customer list view in my local.

**Comment by 0Pranav** [comment]
The best solution to this issue seems to be to remove the default value from `Selling Settings`

**Pull request closed without merging**